bpf/lib/nodeport.h

```diff
@@ -3216,9 +3227,11 @@ static __always_inline int
 handle_nat_fwd_ipv4(struct __ctx_buff *ctx, struct trace_ctx *trace,
 		    __s8 *ext_err)
 {
+	__u32 cb_nat_flags = ctx_load_meta(ctx, CB_NAT_FLAGS);
```

Comment:
To be consistent with `handle_nat_fwd_ipv6()`:

```suggestion
	__u32 cb_nat_flags = ctx_load_and_clear_meta(ctx, CB_NAT_FLAGS);
```